participants/alita.global.md

```diff
@@ -0,0 +1,50 @@
+## Alita Model of Decentralized Supercomputing Network
+## (0) Project Name and Team Slack Handles
+##### project's name: Alita Model of Decentralized Supercomputing Network
+##### Slack handle: jinbin
+## (1) Brief Project Description
+A cloud computing network composed of social idle computing capacity. Committed to building asset-light, low-cost, high-availability computing platform. Alita builds a verifiable computing model based on TEE. It already supports both ARM and X86 servers and edge devices. And it is the world's first MapReduce framework to support Android.
+## (2) Link to Project UI
+https://alita.global http://datadex.trade/
+
+## (3) What does your application/UI do?
+DataDEX base Alita is a data exchange platform based on the Alita network. It uses the technology of DeFi to provide liquidity pricing strategy, maximize the value of data, truly realize data ownership, and provide privacy preserving ability to ensure that data is not collected and only the access permissions are traded.
+
+![avatar](http://datadex.trade/images/dataexchange.PNG) 
+## (4) If your project is using a curated dataset, which one are you using?
+Condensed Movies	A large-scale video dataset, featuring clips from movies with detailed captions.
```

Comment:
I wasn't sure how you're using this dataset.   Will this be available to your users or the public?